src/components/Logo/index.tsx

```diff
@@ -34,5 +60,9 @@ export default function Logo({ srcs, alt, style, ...rest }: LogoProps) {
     )
   }
 
-  return <Slash {...rest} style={{ ...style, color: theme.deprecated_bg4 }} />
+  return (
+    <MissingImageLogo size={size} getFontSize={getFontSize}>
+      {symbol?.toUpperCase().replace('$', '').replace(/\s+/g, '').slice(0, 3)}
```

Comment:
I think the double replace makes this more readable.

Is the `slice` necessary? Does it need to be restricted to `3` characters?

Reply:
yeah we do need the two replaces (i think), not sure how else I'd do this. also only first 3 chars is necessary, as determined with fred, because it looks cramped when more than 3 chars are permitted.

Reply:
Can you leave a comment that "it looks cramped when more than 3 chars are used"? Otherwise a future eng may call this out as a bug.

Reply:
done